src/components/Check/Check.scss

```diff
@@ -49,19 +56,30 @@ $checkBoxHeight: 20px;
 }
 
 .ms-Check-background {
+  font-family: 'FabricMDL2Icons';
+  position: relative;
+
   &:before {
+    content: '\E91F';
+    color: $ms-color-white;
+  }
+
+  &:after {
+    content: '\EA3A';
+    color: $ms-color-neutralTertiaryAlt;
+  }
+
+  &:before,
+  &:after {
+    @include left(50%);
+    @include transform-translateX(-50%);
```

Comment:
for these scenarios, you don't need to make them RTL specific so you can just say left: 50%; you are creating symmetrical padding now, so it will render correctly in LTR/RTL regardless.